### PR TITLE
Add changelog for v1.0.2

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.0.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* 0decedb Update unit tests and disable broken status unit tests.
+* fddc554 Use existing content name if already in snapshot object for static provisioning, otherwise construct new name for content for dynamic provisioning.
+* 96e3bd1 Add `VolumeSnapshotContent` deletion policy.
+* 89a64c9 Fix failed unit tests.
+* ea17039 Add `VolumeSnapshot` and `VolumeSnapshotContent` in Use Protection using Finalizers.
+* 7cd3ef6 Update README to clarify snapshot support status
+* 097a4b8 Use protosanitizer library so secrets will be stripped from the logs.
+* 0cf36ad Adds new reserved prefixed parameter keys which are stripped out of parameter list, and adds deprecation notice for old keys and keep their behavior the same.
+
+# Changelog since v0.4.0
```
Comment (msau42) [inline CHANGELOG-1.0.md]: I think we don't need this since this is the changelog for 1.0.x versions

File: CHANGELOG-1.0.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Changelog since v1.0.1
+
+## Notable Changes
+
+- Cherry picks PR #172: Added extra verification of source PersistentVolumeClaim before creating snapshot.([#175](https://github.com/kubernetes-csi/external-snapshotter/pull/175), [@xing-yang](https://github.com/xing-yang))
+
+# Changelog since v0.4.1
+
+## Breaking Changes
+* c422fbd Switch to use TypedLocalObjectReference in core API.
```
Comment (msau42) [inline CHANGELOG-1.0.md]: In the preview, these don't show up as links. We may need to manually substitute the PR link for these.

File: CHANGELOG-1.0.md (hunk 1 above)
Comment (xing-yang) [reply CHANGELOG-1.0.md]: sure.

File: CHANGELOG-1.0.md (hunk 2 above)
Comment (xing-yang) [reply CHANGELOG-1.0.md]: Ok, I'll look them up.

**Comment by xing-yang** [comment]
@msau42 Addressed your comments.  Thanks.

**Pull request merged**